perf/benchmark/templates/fortio.yaml

```diff
@@ -170,14 +170,13 @@ spec:
 {{ toYaml $.Values.appresources1 | indent 10 }}
 {{- end }}
 ---
-{{- if $.V.inject }}
+{{- if eq $.name "fortioserver" }}
```

Comment:
sorry i forgot where the name is set, how this value is propogated? i think putting this out of fortio.yaml, just a separate mtls.yaml in `templates/` with hardcoded value is just easier for us?